peter_lin/midterm/main.rb

```diff
@@ -0,0 +1,216 @@
+require_relative 'lib/animal'
+require_relative 'lib/client'
+require_relative 'lib/shelter'
+require_relative 'seeds'
+
+s = create_shelter_data()
+
+while true
+  puts '-----------------------------'
+  puts 'Shelter Options'
+  puts '-----------------------------'
+  puts '(1) Create an animal'
+  puts '(2) Create a client'
+  puts "(3) Print all of the shelter\'s animals"
+  puts "(4) Print all of the shelter\'s clients"
+  puts "(5) Client wants to adopt an animal"
+  puts "(6) Client wants to put an animal up for adoption"
+  puts '(7) Client wants play with a pet'
+  puts '(8) Give a toy to a pet'
+  puts '(9) Quit'   
+  puts '-----------------------------'
+
+  print 'Pick an option: '
+
+  option = gets.chomp.to_i
+
+  if option == 1
+    puts 'Creating an animal...'
```

Comment:
suggested refactor: the actions after the user picks an option can be extracted into methods to make this read a little better. for instance

if option == 1
   create_animal
elsif..

just a thought